mlflow/pyfunc/model.py

```diff
@@ -174,6 +174,8 @@ def __init_subclass__(cls, **kwargs) -> None:
             predict_stream_attr = cls.__dict__.get("predict_stream")
             if predict_stream_attr is not None and callable(predict_stream_attr):
                 _check_func_signature(predict_stream_attr, "predict_stream")
+        else:
+            cls.predict._is_pyfunc = True
```

Comment:
Is this testable?

Reply:
To test this we'll need to add a class in mlflow module, do you think we need to?

Reply:
If it's challenging to test this, it's ok.